ledger/participant-state/kvutils/src/main/protobuf/com/daml/ledger/participant/state/kvutils/daml_configuration.proto

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+//
+// Protocol buffer messages used by the participant state key-value utilities
+// for implementing a Daml ledger backed by a key-value store.
+//
+// These messages should only be produced and consumed by the methods in
+// `KeyValueCommitting`, `KeyValueConsumption` and `KeyValueSubmission` objects.
+//
+
+syntax = "proto3";
+
+package com.daml.ledger.participant.state.kvutils;
+
+option java_package = "com.daml.ledger.participant.state.kvutils";
+option csharp_namespace = "Com.Daml.Ledger.Participant.State.KVUtils";
+
+import "google/protobuf/timestamp.proto";
+import "com/daml/ledger/configuration/ledger_configuration.proto";
+
+// Configuration change request to change the ledger configuration.
+message DamlConfigurationSubmission {
```

Comment:
If we move this into _submission.proto_, can we move the other configuration message types into the correct places?

Reply:
The `wire` package contains messages that we only transmit via the network (and not persist to the ledger), hence, `DamlConfigurationSubmission` definitely belongs there.

Reply:
@miklos-da good point, the `DamlConfigurationSubmission` definitely belongs there. 
@SamirTalwar Unfortunately not. `DamlConfigurationSubmission` is indeed only reference from `submission.proto`. The problem is `DamlConfigurationEntry` is used in `state.proto`. `state.proto` is used by `rejection_reason.proto` which is in turn used by `configuration_rejection.proto`. This means we can't merge `configuration.proto` and `configuraiton_rejection.proto` because it generates a circular import.

Reply:
Welp, boo.

In the future perhaps we can decouple these.